doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc

```diff
@@ -78,8 +74,8 @@ ifdef::PAM[]
 +
 .. *Dialect*: Enter the Hibernate dialect for the server, depending on the server type:
 +
-*** `org.hibernate.dialect.MySQL5InnoDBDialect` (used for MySQL and MariaDB)
-*** `org.hibernate.dialect.PostgreSQL82Dialect`
+*** `org.hibernate.dialect.MySQL5InnoDBDialect` (used for MySQL version 5.7 and for MariaDB)
+*** `org.hibernate.dialect.MySQL8Dialect` (used for MySQL version 8)
```

Comment:
IIRC Mariadb has it own dialect as well, 
for postgreeSQL use the dialect for version 10.

Reply:
I could not find a MariaDB dialect and a Postrges 10 dialect; as ths issue arose I'll await answers at https://issues.redhat.com/browse/BXMSDOC-6014 and put in the new dialect list in this PR